Source/DafnyDriver/Commands/CommandRegistry.cs

```diff
@@ -67,7 +67,12 @@ static class CommandRegistry {
       if (!allowHidden) {
         option.IsHidden = false;
       }
+      option.AllowMultipleArgumentsPerToken = true;
     }
+    RunCommand.Inputs.AllowMultipleArgumentsPerToken = false;
+    CommonOptionBag.Libraries.AllowMultipleArgumentsPerToken = false;
```

Comment:
Why are some of these false? Is that because they allow multiple values as inputs? Can we add that check on line 70 instead?

Reply:
No - because all of this setting is false by default; besides why check them on each iteration of the loop when they can just be set afterwards. 

These three are false because that is the current behavior.
We could say that they take a comma-separated list instead, as I've proposed to Aaron for --log-format. We have already implemented comma-separated-lists for --libraries, but not restricted its use. I can't see restricting --input as it is used in dafny run

Reply:
Maybe you could check whether the .Arity is `ZeroOrMore` or `OneOrMore`, and if so on that keep `AllowMultipleArgumentsPerToken` at false?

Reply:
Made this change.